.github/workflows/ci.yaml

```diff
@@ -0,0 +1,86 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  pull_request_test:
+    if: ${{ github.event.issue.pull_request }}
+    timeout-minutes: 180
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+          java-package: jdk
+          architecture: x64
+      - name: Cache local maven repository
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.m2/repository/
+            !~/.m2/repository/org/apache/servicecomb
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+      - name: Show the first log message
+        run: git log -n1
+      - name: Build and Test and Install Docker
+        if: ${{ success() }}
+        run: ./mvnw clean install -B -Pdocker
+      - name: Upload unit test results when the previous step of a job fails
+        if: ${{ failure() }}
```

Comment:
Why we do this if the tests are failed?

Reply:
After the test fails, the unit test report will be uploaded to Github for easy download through the Github web interface

Reply:
It makes sense.